Update main.py

yomisimie opened the pull request:
I tried to add in configuration a key called `dimensions` but it crashed the script, I'm not that good with python. I added this here and works fine, can you add a configuration only for grids type?

luthor112 commented:
If you want to specify the "dimensions" parameter for girds, just add it to the "grids_config" object in your config.json.

yomisimie commented:
> If you want to specify the "dimensions" parameter for girds, just add it to the "grids_config" object in your config.json.

WTF, I added this because initially when I added the parameter it crashed the plugin. Made sense to add it there considering you are making query params from that list, now it works. Thanks!
